remake.yml

```diff
@@ -32,18 +33,40 @@ targets:
   output:
     command: produceAnalysesOutputs(metRates)
 
-  tables:
-    command: makeTables(dataTable1=metRates, pathToDataTable2='output/RDatafiles/analyses.RData')
-  
+  output/data/table1.csv:
+    command: makeTable1(metRates, target_name)
+
+  output/data/table2.csv:
+    command: makeTable2(output, target_name)
+
   numbersList:
-    command: extractNumbersList(metRates, 'output/RDatafiles/analyses.RData')
+    command: extractNumbersList(metRates, output)
```

Comment:
note we're passing in the R object, instead of loading it from file within the fucntion